manifest.json

```diff
@@ -1,14 +1,14 @@
-
 {
-  "manifest_version": 2,
   "name": "WebAssist",
   "author": "Microsoft Edge Extension Developer",
   "version": "1.1",
+  "manifest_version": 2,
   "description": "Assistance for your web browser",
   "permissions": [
-    "<all_urls>",
-    "activeTab",
-    "tabs"
+    "*",
```

Comment:
This needs to be taken out - we want all URL's to be accessed, but not all permissions